src/kartograf/tests/test_atom_mapping_comparer.py

```diff
@@ -0,0 +1,89 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/kartograf
+
+import pytest
+from gufe import LigandAtomMapping
+from kartograf.mapping_metrics.metrics_mapping_comparisons import (
+        jaccard_score
+)
+
+from .conf import benzene_benzene_mapping, benzene_benzene_empty_mapping
```

Comment:
if you renamed `conf.py` to `conftest.py` you could use these fixtures without import